src/hardware/vga_draw.cpp

```diff
@@ -2054,6 +2054,13 @@ ImageInfo setup_drawing()
 		return pcjr_or_tga();
 	};
 
+	// As far as I can tell, all Tandy modes have a height of 200.
+	// Some games (ex. Impossible Mission II) fiddle with vga.other.vdend
+	// The result of this should be rendering a short (by height) image in the horizonal center
+	// With black on the top/bottom
```

Comment:
minor: join the above two lines into a single sentence, period at the end.

Reply:
fixed